app/models/google_event.rb

```diff
@@ -1,4 +1,16 @@
 class GoogleEvent
+  class InvalidParamsException < Exception; end
```

Comment:
Don't use `Exception` as base class, either use `StandardError` or `RuntimeError`. More here: https://robots.thoughtbot.com/rescue-standarderror-not-exception